### PR TITLE
[J4.1] Better "Your Modules" Article

### PR DESCRIPTION
**Pull request by Razzo1987**
Pull Request for Issue #36851 .

### Summary of Changes
Better "Your Modules" article making the article consistent with J4

### Testing Instructions
Install sample data and check the Your Modules article and the modules in the back-end

### Actual result BEFORE applying this Pull Request
---

### Expected result AFTER applying this Pull Request
image

### Documentation Changes Required
Language change
For the translator check the name of the modules in your language

### CODE REVIEW

File: administrator/language/en-GB/plg_sampledata_blog.ini [hunk 1]
```diff
@@ -25,7 +25,7 @@ PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_3_INTROTEXT="<p>Your home page is
...
...ur site has some commonly used modules already preconfigured. These include:</p><ul><li>Image (type: Custom), which holds the image beneath the menu. This is a Custom module that you can edit to change the image.</li><li>Popular Tags (type: Tags - Popular), which will appear if you use tagging on your articles. Enter a tag in the Tags field when editing.</li><li>Older Posts (type: Articles - Category), which lists out articles by categories.</li><li>Syndication (type: Syndication Feeds), which allows your readers to read your posts in a news reader.</li><li>Login Form (type: Login), which allows your users to access the website.</li></ul><p>Each of these modules has many options which you can experiment with in the Module Manager in your site Administrator. When you are logged in you can also select the edit icon in the top right corner which will take you to an edit screen for that module. Always be sure to save and close any module you edit.</p><p>Joomla! also includes many other modules you can incorporate in your site. As you develop your site you may want to add more modules that you can find at the <a href="https://extensions.joomla.org/">Joomla Extensions Directory.</a></p>"
```
Comment (brianteeman) [inline administrator/language/en-GB/plg_sampledata_blog.ini]:
```suggestion
PLG_SAMPLEDATA_BLOG_SAMPLEDATA_CONTENT_ARTICLE_4_INTROTEXT="<p>Your site has some commonly used modules already preconfigured. These include:</p><ul><li>Image (type: Custom), which holds the image beneath the menu. This is a Custom module that you can edit to change the image.</li><li>Popular Tags (type: Tags - Popular), which will appear if you use tagging on your articles. Enter a tag in the Tags field when editing.</li><li>Older Posts (type: Articles - Category), which lists out articles by categories.</li><li>Syndication (type: Syndication Feeds), which allows your readers to read your posts in a news reader.</li><li>Login Form (type: Login), which allows your users to access restricted areas of the website.</li></ul><p>Each of these modules has many options which you can experiment with in the Module Manager in your site Administrator. When you are logged in you can also select the edit icon in the top right corner which will take you to an edit screen for that module. Always be sure to save and close any module you edit.</p><p>Joomla! also includes many other modules you can incorporate in your site. As you develop your site you may want to add more modules that you can find at the <a href="https://extensions.joomla.org/">Joomla Extensions Directory.</a></p>"
```

Comment (Razzo1987) [reply administrator/language/en-GB/plg_sampledata_blog.ini]: Note:
Change:
`Login Form (type: Login), which allows your users to access the website.`
in
`Login Form (type: Login), which allows your users to access restricted areas of the website.`

OK for me

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 283497d3fb8f4b028ef4dadf7f15efbf64858391<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36865">issues.joomla.org/tracker/joomla-cms/36865</a>.</sub>

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on 283497d3fb8f4b028ef4dadf7f15efbf64858391<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36865">issues.joomla.org/tracker/joomla-cms/36865</a>.</sub>

**Comment by chmst** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/36865">issues.joomla.org/tracker/joomla-cms/36865</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Comment by bembelimen** [comment]
Thx

**Pull request merged**